kamailio/db_scripts/mysql-specific

```diff
@@ -6,6 +6,8 @@ EOF
 }
 
 sql_setup() {
+cat << EOF
+EOF
```

Comment:
@sergey-safarov taking a 2nd look at this. `sql_setup` is to actually do the setup, its not meant to capture anything. you may add the mysql and/or postgres command to initialize the db like we do in db_kazoo where we call `KazooDB.

Reply:
@lazedo yes i understand.
Now no need to do anything. But we cannot leave this function empty. Because sh cannot parse functions.
You can try execute on current master this command, wnd will get error
```
[build@aeyears db_scripts]$ DB_ENGINE=postgres ./create-kazoodb-sql.sh
./postgres-specific: line 9: syntax error near unexpected token `}'
[build@aeyears db_scripts]$ 
```

Reply:
Updated functions is outputs to terminal empty string
same as `echo`

Reply:
ok, thanks for the feedback